scripts/roots_of_quadratic.py

```diff
@@ -0,0 +1,8 @@
+#roots of quadratic equation in the form of ax^2 + bx + c = 0
```

Comment:
Trivial: Could you move this comment inside the function and add it as a docstring instead?